Makefile

```diff
@@ -1,13 +1,22 @@
 TEST_TARGET := ./tests/
+KNIFE := knife
+
+build-knife:
+	ansible-bender build ./knife.yml registry.fedoraproject.org/fedora:29 $(KNIFE)
 
 check:
 	PYTHONPATH=$(CURDIR) pytest-3 -v $(TEST_TARGET)
 
 check-pypi-packaging:
-	podman run --rm -ti -v $(CURDIR):/src -w /src registry.fedoraproject.org/fedora:29 bash -c '\
+	podman run --rm -ti -v $(CURDIR):/src -w /src $(KNIFE) bash -c '\
 		pip3 install . \
-		&& ab --help \
-		&& ab build --help'
+		&& ansible-bender --help \
+		&& ansible-bender build --help \
+		&& pip3 show ansible-bender \
+		## python3 ./setup.py sdist \
+		&& twine check ./dist/* \
```

Comment:
What command creates the dist? I don't see it. `twine` checks pre-built dists...

Reply:
I accidentally put `## py...` in there, should be okay now.

So the check-packaging target works very well now, I think I'm good overall. I don't want to play with python packaging in near (distant really) future. I like the progress though, the fact that one can define packaging in setup.cfg is such a good move.

Thank you so much for guiding me through these unknown lands. I really learned something.

Reply:
:+1: